src/shared/components/navbar/Navbar.scss

```diff
@@ -1,15 +1,20 @@
 @import 'styles/mixins';
+@import 'styles/media-queries';
 
 .navbar {
   display: grid;
-  grid-template-columns: 1fr 1200px 1fr;
+  grid-template-columns: 1fr 10fr 1fr;
```

Comment:
1200px is width of container fluid for desktop devices - all content on home page and in other pages must be 1200  for desktop